cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,52 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.2.2 <http://docs.cask.co/cdap/3.2.2/index.html>`__
+=============================================================
+
+Improvement
+-----------
+
+- `CDAP-4133 <https://issues.cask.co/browse/CDAP-4133>`__ -
+  Added ability to get the live-info for the CDAP AppFabric system service.
+
+Bug Fixes
+---------
+
+- `CDAP-3644 <https://issues.cask.co/browse/CDAP-3644>`__ -
+  Fixed a problem with upgrading the metadata system datasets.
+
+- `CDAP-4067 <https://issues.cask.co/browse/CDAP-4067>`__ -
+  Fixed an issue where socket connections to the TransactionManager were not being closed.
+
+- `CDAP-4092 <https://issues.cask.co/browse/CDAP-4092>`__ -
+  Fixes an issue that causes worker threads to go into an infinite recursion while
+  exceptions are being thrown in channel handlers.
+
+- `CDAP-4093 <https://issues.cask.co/browse/CDAP-4093>`__ -
+  Fixed a problem with installation documentation for Cloudera Manager and Apache Ambari.
+
+- `CDAP-4097 <https://issues.cask.co/browse/CDAP-4097>`__ -
+  Fixed a problem with the AbstractDatasetProvider throwing exceptions and polluting the logs.
+
+- `CDAP-4119 <https://issues.cask.co/browse/CDAP-4119>`__ -
+  Fixed a problem where, when the CDAP Master switched from active to standby, that 
+  any programs that were running were marked as failed.
+
+- `CDAP-4275 <https://issues.cask.co/browse/CDAP-4275>`__ -
+  Fixed a problem with searching CDAP entities based on tags where the entity has multiple tags.
```

Comment:
Let's remove this issue. Metadata search does not work as expected. We need to add CDAP-4393 as a known issue instead.